python/cuda_parallel/cuda/parallel/experimental/_iterators.py

```diff
@@ -433,6 +417,25 @@ def transform_dereference(it_state_ptr):
         # ATTENTION: op_caller here (see issue #3064)
         return op_caller(source_dereference(it_state_ptr))
 
+    prefix = f"transform_{it.prefix}_{op.__name__}"
+    ltoirs =  [
+        *it.ltoirs,
```

Comment:
It's much better having that code here!

Tiny nit: `ltoirs = it.ltoirs + [` seems a little more direct.

Reply:
Changed it to that